linux/scripts/upload-to-debian.sh

```diff
@@ -73,56 +73,73 @@ if ! git diff --quiet; then
     builder_die "You have changed files in your git working directory. Exiting."
 fi
 
-builder_heading "Fetching latest changes"
-git fetch -p origin
-stable_branch=$(git branch -r | grep origin/stable- | sort | tail -1)
-stable_branch=${stable_branch##* }
-# Checkout stable branch so that `scripts/debian.sh` picks up correct version
-git checkout "${stable_branch#origin/}"
-git pull origin "${stable_branch#origin/}"
-
-if git branch -r | grep origin/beta; then
+if [[ "$(cat TIER.md)" == "beta" ]]; then
     ISBETA=true
 else
     ISBETA=false
 fi
 
+builder_heading "Fetching latest changes"
+git fetch -p origin
+if ${ISBETA}; then
```

Comment:
Here you are switching between beta and stable, it seems, but on line 142 you seem to be switching between beta and alpha?